Source/ACE.Database/WorldDatabase.cs

```diff
@@ -250,6 +250,18 @@ public void CacheAllWeenies()
             }
         }
 
+        public String GetScrollBySpellID(uint spellID)
```

Comment:
We're going to need to add some sort of caching mechanism here.

The way you have it written, it appears that every request will hit the database.